SillyBusinessInc/Moldbreaker
# Feat/#213 enemy healthbar

## Purpose of this PR:
Enemy has a healthbar now
## How to test:
Add the enemyhealthbar prefab in a enemy, add the enemy in the enemy Base field in the healthbar script. Attack the player, healthbar should work

### links: 
[Ticket](https://github.com/SillyBusinessInc/SillyBusinessGame/issues/213)
